src/bom/bitcode.rs

```diff
@@ -345,6 +327,25 @@ fn build_bitcode_arguments(chan : &mut mpsc::Sender<Option<Event>>,
                 orig_target = Some(target);
                 bcgen_op.set_output_file(&FileArg::loc(target_path));
             }
+        } else {
+            // This is not an output specifying argument.
+            if arg.to_string_lossy().starts_with("-") {
+                bcgen_op.push_arg(arg);
+                if clang_support::next_arg_is_option_value(arg) {
+                    match it.next() {
+                        Some(val) => { bcgen_op.push_arg(val); }
+                        None => {
+                            return Err(anyhow::Error::new(
```

Comment:
This `return Err(anyhow::Error(...))` pattern is captured nicely by the `bail!` macro: https://docs.rs/anyhow/latest/anyhow/macro.bail.html

Reply:
Nice: updated.